api/src/handlers.rs

```diff
@@ -403,6 +447,7 @@ pub fn start_rest_apis<T>(
 
 		let route_list = vec!(
 			"get /".to_string(),
+			"get /block".to_string(),
```

Comment:
I would use plural. `chain` and `pool` are singular because there's ever only one, but note `utxos` and `sumtrees`.